projects/plugins/jetpack/tests/e2e/jest-playwright.config.js

```diff
@@ -3,15 +3,11 @@
  * https://playwright.dev/docs/api/class-browsertype?_highlight=launch#browsertypelaunchoptions
  */
 
-const { CI, E2E_DEBUG, HEADLESS, SLOWMO } = process.env;
-let executablePath = '';
-
-if ( ! CI ) {
-	executablePath = '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
-}
+let { E2E_DEBUG, HEADLESS, SLOWMO } = process.env;
 
 if ( E2E_DEBUG ) {
 	process.env.DEBUG = 'pw:browser|api|error';
```

Comment:
I think this doesn't really work. This code works in a `jest`'s setup process, while tests are running in separate processes. And I believe the environment is not shared between these processes.

Try accessing `DEBUG` value in one of the tests.

Reply:
Actually it works. Not sure how, they're probably shared or inherited.